sixMans/sixMans.py

```diff
@@ -11,28 +11,27 @@
 
 from .game import Game
 from .queue import SixMansQueue
+from .strings import Strings
 
-DEBUG = False
+DEBUG = True
```

Comment:
This should be changed now that it's going to master

Reply:
Good catch, thank you!